packages/suite/src/views/settings/SettingsCoins/SettingsCoins.tsx

```diff
@@ -81,18 +87,27 @@ const getDiscoveryButtonAnimationConfig = (isConfirmed: boolean): MotionProps =>
 
 export const SettingsCoins = () => {
     const { firmwareTypeBannerClosed } = useSelector(selectSuiteFlags);
-
     const isDiscoveryButtonVisible = useRediscoveryNeeded();
     const { mainnets, testnets, enabledNetworks, setEnabled } = useEnabledNetworks();
-    const deviceSupportedNetworks = useSelector(selectDeviceSupportedNetworks);
-    const supportedEnabledNetworks = enabledNetworks.filter(enabledNetwork =>
-        deviceSupportedNetworks.includes(enabledNetwork),
-    );
-
+    const deviceSupportedNetworkSymbols = useSelector(selectDeviceSupportedNetworks);
+    const deviceModel = useSelector(selectDeviceModel);
     const { device } = useDevice();
     const dispatch = useDispatch();
     const { isDiscoveryRunning } = useDiscovery();
 
+    const supportedEnabledNetworks = enabledNetworks.filter(enabledNetwork =>
+        deviceSupportedNetworkSymbols.includes(enabledNetwork),
+    );
+
+    const getNetworks = (networks: Network[], getUnsupported = false) =>
+        networks.filter(
+            ({ symbol }) => getUnsupported !== deviceSupportedNetworkSymbols.includes(symbol),
+        );
```

Comment:
You are using this + `deviceSupportedNetworkSymbols` twice, it might make sense to unify it. But it's not necessary. Up to you.